src/stream-analytics/azext_stream_analytics/generated/_params.py

```diff
@@ -0,0 +1,434 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import (
+    get_default_location_from_resource_group,
+    validate_file_or_dict
+)
+from azext_stream_analytics.action import (
+    AddIdentity,
+    AddTransformation,
+    AddJobStorageAccount
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('stream-analytics job list') as c:
+        c.argument('expand', type=str, help='The $expand OData query parameter. This is a comma-separated list of '
+                   'additional streaming job properties to include in the response, beyond the default set returned '
+                   'when this parameter is absent. The default set is all streaming job properties other than '
+                   '\'inputs\', \'transformation\', \'outputs\', and \'functions\'.')
+        c.argument('resource_group_name', resource_group_name_type)
+
+    with self.argument_context('stream-analytics job show') as c:
+        c.argument('expand', type=str, help='The $expand OData query parameter. This is a comma-separated list of '
+                   'additional streaming job properties to include in the response, beyond the default set returned '
+                   'when this parameter is absent. The default set is all streaming job properties other than '
+                   '\'inputs\', \'transformation\', \'outputs\', and \'functions\'.')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('job_name', options_list=['--job-name', '-n'], type=str, help='The name of the streaming job.')
+
+    with self.argument_context('stream-analytics job create') as c:
+        c.argument('if_match', type=str, help='The ETag of the streaming job. Omit this value to always overwrite the '
+                   'current record set. Specify the last-seen ETag value to prevent accidentally overwriting '
+                   'concurrent changes.')
+        c.argument('if_none_match', type=str, help='Set to \'*\' to allow a new streaming job to be created, but to '
+                   'prevent updating an existing record set. Other values will result in a 412 Pre-condition Failed '
+                   'response.')
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('job_name', options_list=['--job-name', '-n'], type=str, help='The name of the streaming job.')
```

Comment:
I previous version, job_name has options ['--name', '-n'] please keep them
```suggestion
        c.argument('job_name', options_list=['--name', '-n'], type=str, help='The name of the streaming job.')
```

Reply:
It will cause breaking change if we loose --name option